runtime/basilisk/src/assets.rs

```diff
@@ -208,14 +208,14 @@ impl pallet_xyk::Config for Runtime {
 	type AssetPairAccountId = AssetPairAccountId<Self>;
 	type Currency = Currencies;
 	type NativeAssetId = NativeAssetId;
-	type WeightInfo = weights::xyk::BasiliskWeight<Runtime>;
+	type WeightInfo = weights::pallet_xyk::BasiliskWeight<Runtime>;
 	type GetExchangeFee = ExchangeFee;
 	type MinTradingLimit = MinTradingLimit;
 	type MinPoolLiquidity = MinPoolLiquidity;
 	type MaxInRatio = MaxInRatio;
 	type MaxOutRatio = MaxOutRatio;
 	type OracleSource = XYKOracleSourceIdentifier;
-	type CanCreatePool = pallet_lbp::DisallowWhenLBPPoolRunning<Runtime>;
+	type CanCreatePool = hydradx_adapters::xyk::AllowPoolCreation<Runtime, AssetRegistry>;
```

Comment:
hm.. suspicious.  what happened here? double check

Reply:
yeah, I think it should be fine, it has only strange naming, I changed it to disallow specific xyk pool creation.

As I was the one who changed it, please someone else do a double check then